pkg/cloudprovider/providers/azure/azure_client.go

```diff
@@ -17,12 +17,15 @@ limitations under the License.
 package azure
 
 import (
+	"context"
+	"net/http"
 	"time"
 
 	"github.com/Azure/azure-sdk-for-go/arm/compute"
 	"github.com/Azure/azure-sdk-for-go/arm/disk"
 	"github.com/Azure/azure-sdk-for-go/arm/network"
 	"github.com/Azure/azure-sdk-for-go/arm/storage"
+	computepreview "github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2017-12-01/compute"
```

Comment:
any reason why rename it? This rename causes lots of diffs.

Reply:
There is already an existing package with name `compute`: "github.com/Azure/azure-sdk-for-go/arm/compute"